TreeTracker/UI/Views/Home/Components/UploadsButton.swift

```diff
@@ -60,12 +66,15 @@ private extension UploadsButton {
     }
 
     func updateState() {
-        if isEnabled {
-            backgroundColor = Asset.Colors.secondaryOrangeLight.color
-            tintColor = .white
-        } else {
+        if !isEnabled {
             backgroundColor = Asset.Colors.grayLight.color.withAlphaComponent(0.2)
             tintColor = Asset.Colors.grayLight.color
+        } else if isHighlighted {
+            backgroundColor = Asset.Colors.secondaryOrangeDark.color
+            tintColor = .white
+        } else {
+            backgroundColor = Asset.Colors.secondaryOrangeLight.color
+            tintColor = .white
```

Comment:
Since `isEnabled` and `isHighlighted` are different, you can put them in separate conditional statements
something like
```
if !isEnabled {
    ...
}

if isHighlighted {
    ...
} else {
    ...
}
```

Reply:
Great! I'll make this change